schema.ts

```diff
@@ -1016,7 +1016,7 @@ export type InferrableTypes = boolean | number | string | object | boolean[] | n
  * inputs, it may be useful to us this helper to sniff the return value and generate a basic
  * inferred schema from it.
  *
- * This utility does NOT attempt to determine {@link id} or {@link primary} attributes for
+ * This utility does NOT attempt to determine {@link idProperty} or {@link displayProperty} attributes for
```

Comment:
the old values were ambiguous. https://coda.io/packs/build/latest/reference/sdk/functions/generateSchema/

it's more precise to prefix with a classname.